src/idiom250.fuzion

```diff
@@ -0,0 +1,4 @@
+x := m.values
+
+x.drop (random.next_i32 x.count)
+ .head
```

Comment:
The idioms says _Choose a value x from map m. m must not be empty. Ignore the keys._, so this must be
```
x := m.values
       .drop (random.next_i32 m.size)
       .head
```
BTW, here is an inconsistency in our base library: `Sequence.count` vs. `map.size` that should be fixed some time.